src/internal/serialization/protobuf/verify/VerifyBlockchain.hpp

```diff
@@ -26,6 +26,9 @@ auto BlockchainBlockHeaderAllowedBlockchainBlockLocalData() noexcept
     -> const VersionMap&;
 auto BlockchainBlockHeaderAllowedEthereumBlockHeaderFields() noexcept
     -> const VersionMap&;
+// TODO: Casper. Defining a protobuf serialization for Casper block headers
+// and transactions is needed in the stage where opentxs is storing
+// those transactions itself in the blockchain database.
```

Comment:
maybe add JIRA no. or assign to the existing ones in the comment

Reply:
Yet we don't have any tasks defined for Casper-OT integration part. We will be defining them from scratch later anyway. If we created it now, it could get lost in the JIRA depths.